app/Http/Controllers/FrontendUserController.php

```diff
@@ -96,4 +98,18 @@ public function updateProfilePicture(Request $request)
         $profilePictureResponse = UserBackend::updateProfilePicture($avatar);
         return response()->json($profilePictureResponse);
     }
+
+    public function searchUser(Request $request) {
+        try {
+            $userSearchResponse = UserBackend::searchUser($request['searchQuery']);
+            $userSearchResponse->appends(['searchQuery' => $request['searchQuery']]);
```

Comment:
I recommend to pass the variable searchQuery explicit to the view below instead of adding it to this collection. (or is this for something useful?)

Reply:
This is needed for the pagination. Otherwise the links throw away the search parameters, which will result in a bad request. It's the recommended practice from the docs: [https://laravel.com/docs/8.x/pagination#appending-to-pagination-links](https://laravel.com/docs/8.x/pagination#appending-to-pagination-links)